### ISSUE TITLE
ReadIn PySpice Schematics

### ISSUE THREAD

**Issue by GProtoZeroW** [mph-/lcapy#13]
I am wanting to try using lcapy to read back a netlist from either skidl (https://github.com/xesscorp/skidl/blob/master/examples/spice-sim-intro/spice-sim-intro.ipynb) or PySpice (https://pyspice.fabrice-salvaire.fr/examples/filter/rlc-filter.html) into a Schematic. I tried with the above PySpice RLC filter but with no luck. My self and Skidl's author have been looking for a means to translate the netlist level stuff we have been doing in python to schematics that don't generate a rats nest. We have been looking at  SchemeDraw. However, after looking at your lib at first glance, it may be able to do what we have been looking for. If you can give some of the examples from Skidl and PySpice a try that would be greatly appreciated

**Comment by Geocali** [comment]
@GProtoZeroW Did you find a way to do this ? This could be quite interesting

**Comment by natask** [comment]
I think so as well.

**Comment by mph-** [comment]
I was hoping that someone else was going to have a go at this.  I might have a go over my upcoming summer holiday.

**Comment by joaoantoniocardoso** [comment]
It would be wonderful for both communities. 

I hadn't jumped into the PySpice nor lcapy internals, but I suppose we would have to create an interface between the `PySpice.Circuit` the `lcapy.Circuit` classes.

Have you tried anything?

**Comment by mph-** [comment]
No, I have not tried this.  My current TODO list is long and work commitments get in the way...  I was hoping someone else would have a go at this.

**Comment by joaoantoniocardoso** [comment]
That's fine, I didn't mean to add any pressure, I'm willing to help in any part that is in the way of my current use case, and unfortunately, this one isn't, although I found it interesting, so maybe next year, I can come back to this issue.

Not an expert here, but to incentivize the community, I've seen the use of the tag _wanting help_ in other repositories, to indicate someone from the community should get their hands on it.
